src/components/pages/BrandingPage/index.jsx

```diff
@@ -85,7 +85,7 @@ class BrandingPage extends React.Component {
                     >
                       <MDBIcon icon="download" className="pr-2" />
                       .svg
-                    </MDBBtn>
+                    </MDBBtn>jo 
```

Comment:
typed in the wrong tab?

Reply:
Oh damn. This was just a test I used to see if the rigth component renders.